internal/config/config.go

```diff
@@ -272,18 +299,19 @@ func (c *Config) SetJiraToken(token *oauth1.Token) {
 
 // configFile is a serializable representation of the current Viper configuration.
 type configFile struct {
-	LogLevel    string        `json:"log-level" mapstructure:"log-level"`
-	GithubToken string        `json:"github-token" mapstructure:"github-token"`
-	JiraUser    string        `json:"jira-user" mapstructure:"jira-user"`
-	JiraToken   string        `json:"jira-token" mapstructure:"jira-token"`
-	JiraSecret  string        `json:"jira-secret" mapstructure:"jira-secret"`
-	JiraKey     string        `json:"jira-private-key-path" mapstructure:"jira-private-key-path"`
-	JiraCKey    string        `json:"jira-consumer-key" mapstructure:"jira-consumer-key"`
-	RepoName    string        `json:"repo-name" mapstructure:"repo-name"`
-	JiraURI     string        `json:"jira-uri" mapstructure:"jira-uri"`
-	JiraProject string        `json:"jira-project" mapstructure:"jira-project"`
-	Since       string        `json:"since" mapstructure:"since"`
-	Timeout     time.Duration `json:"timeout" mapstructure:"timeout"`
+	LogLevel      string        `json:"log-level" mapstructure:"log-level"`
+	GithubToken   string        `json:"github-token" mapstructure:"github-token"`
+	JiraUser      string        `json:"jira-user" mapstructure:"jira-user"`
+	JiraToken     string        `json:"jira-token" mapstructure:"jira-token"`
+	JiraSecret    string        `json:"jira-secret" mapstructure:"jira-secret"`
+	JiraKey       string        `json:"jira-private-key-path" mapstructure:"jira-private-key-path"`
+	JiraCKey      string        `json:"jira-consumer-key" mapstructure:"jira-consumer-key"`
+	RepoName      string        `json:"repo-name" mapstructure:"repo-name"`
+	JiraURI       string        `json:"jira-uri" mapstructure:"jira-uri"`
+	JiraProject   string        `json:"jira-project" mapstructure:"jira-project"`
+	JiraComponent string        `json:"jira-component" mapstructure:"jira-component"`
```

Comment:
```suggestion
	JiraComponents []string        `json:"jira-component" mapstructure:"jira-component"`
```

Fix logic throughout